docs/src/components/appContext.js

```diff
@@ -0,0 +1,87 @@
+// @flow strict
+import React, { useEffect, useState, type Node } from 'react';
+import createHydra, { type Hydra } from './createHydra.js';
+import useLocalStorage from './useLocalStorage.js';
+
+const localStorageOrganizedByKey = 'gestalt-sidebar-organized-by';
+const propTableVariantKey = 'gestalt-propTable-variant';
+const localStorageColorSchemeKey = 'gestalt-color-scheme';
+const localStorageTextDirectionKey = 'gestalt-text-direction';
+
+type SidebarOrganisedBy = 'categorized' | 'alphabetical';
+type PropTableVariant = 'collapsed' | 'expanded';
+type ColorScheme = 'light' | 'dark' | 'userPreference';
+type DirectionScheme = 'ltr' | 'rtl';
+
+export type AppContextType = {|
+  isSidebarOpen: boolean,
```

Comment:
@AlbertCarreras This means we now have 1 big context object which contains all context options instead of more dedicated context.

The pros right now is that we have it all in 1 location but the downside is that this will most likely grow into an even bigger object over time, a bit like legacyContent in pinterest's codebase.